int64 plugin_uid

---

# return call status
#   0: success 
#  -1: error
int16 code     
string message
